homeassistant/components/accuweather/system_health.py

```diff
@@ -0,0 +1,28 @@
+"""Provide info to system health."""
+from accuweather.const import ENDPOINT
+
+from homeassistant.components import system_health
+from homeassistant.core import HomeAssistant, callback
+
+from .const import COORDINATOR, DOMAIN
+
+
+@callback
+def async_register(
+    hass: HomeAssistant, register: system_health.SystemHealthRegistration
+) -> None:
+    """Register system health callbacks."""
+    register.async_register_info(system_health_info)
+
+
+async def system_health_info(hass):
+    """Get info for the info page."""
+    remaining_requests = None
+
+    for entry in hass.data[DOMAIN].values():
+        remaining_requests = entry[COORDINATOR].accuweather.requests_remaining
```

Comment:
Not sure of the purpose of this.

The next lines of code only use the final value.

So could you just get the final value without needing the for each?

Reply:
Fixed